src/main/java/domain/Cars.java

```diff
@@ -0,0 +1,35 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(UserInput userInput) {
+        int numbersOfCar = userInput.getNumbersOfCar();
+        List<Car> cars = createCars(numbersOfCar);
+        this.cars = cars;
+    }
```

Comment:
```suggestion
    public Cars(UserInput userInput) {
        this.cars = createCars(userInput.getNumbersOfCar());
    }
```

가독성이 크게 떨어지지 않는다면 지역변수 할당을 최소화 하는 것도 좋을 것 같아요!
(가독성 확보를 위해 종구님이 해주신 방법이 좋을 때도 있어요!!)